<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBAgreementTypeEditor</class>
 <widget class="QDialog" name="RBAgreementTypeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblQuotaStatusModifier">
     <property name="text">
      <string>&amp;Модификатор статуса</string>
     </property>
     <property name="buddy">
      <cstring>cmbQuotaStatusModifier</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cmbQuotaStatusModifier">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Не меняет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Отменено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ожидание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Активный талон</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Талон для заполнения</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Заблокированный талон</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Отказано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Необходимо согласовать дату обслуживания</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Дата обслуживания на согласовании</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Дата обслуживания согласована</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Пролечен</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Обслуживание отложено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Отказ пациента</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Импортировано из ВТМП</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RBAgreementTypeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBAgreementTypeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
